classes/corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize[0..9]
```

Comment:
## * [OPTIONAL]
* I think you can make your range `[0..10]` since its say Make sure that the name has a maximum of 10 characters. If it's longer it should trim the word.